# Represents a maliput::api::SegmentId.
string id
